src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -903,9 +913,41 @@ private static bool HasImplicitNullableConversion(TypeSymbol source, TypeSymbol
                 return true;
             }
 
+            if (HasImplicitTupleConversion(unwrappedSource, unwrappedDestination, ref useSiteDiagnostics))
+            {
+                return true;
+            }
+
             return false;
         }
 
+        private bool HasImplicitTupleConversion(TypeSymbol source, TypeSymbol destination, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
+        {
+            if (!source.IsTupleType || !destination.IsTupleType)
```

Comment:
Added support for underlying types, but we should consider how deep the support for mixing tuples and underlying types need to go.
I think we should not produce underlying types from the binder, and perhaps we should not flow them into semantic analysis from public APIs either.